packages/twenty-chrome-extension/src/manifest.ts

```diff
@@ -48,9 +44,12 @@ export default defineManifest({
     },
   ],
 
-  permissions: ['activeTab', 'storage', 'identity', 'sidePanel'],
+  permissions: ['activeTab', 'storage', 'identity', 'sidePanel', 'cookies'],
 
-  host_permissions: host_permissions,
+  // setting host permissions to all http connections will allow
+  // for people who host on their custom domain to get access to
+  // extension instead of white listing individual urls
+  host_permissions: ['https://*/*', 'http://*/*'],
```

Comment:
Seems like it defeats the purpose of permissions, can't we make it dynamic or have the self-hosting user recompile the extension ?

Reply:
Not realistic to expect self-hosting users to recompile unfortunately, I think most users don't care and prefer the simplest solution which is this one (code is open source / running locally so you could see if it was doing something wrong). I had looked but didn't find any other option.